pkg/repo/db.go

```diff
@@ -22,7 +22,7 @@ var (
 func GetDB() *gorm.DB {
 	singletonOnce.Do(func() {
 		db, err := gorm.Open(config.Config.DBDriver, config.Config.DBConnectionStr)
-		if err != nil {
+		if err != nil && config.Config.DBConnectionDebug {
 			logrus.WithField("err", err).Fatal("failed to connect to db")
 		}
```

Comment:
It seems to me that you want `config.Config.DBConnectionDebug` to gate the output of `err` field, instead of the whole log message.

Is the change below what you intended to make?
```
if err != nil {
  if config.Config.DBConnectionDebug {
    logrus.WithField("err", err).Fatal("failed to connect to db")
  } else {
    logrus.Fatal("failed to connect to db")
  }
}
```